### PR TITLE
Eric Douglass user info

### PR DESCRIPTION
**Pull request by dougli1sqrd**

### CODE REVIEW

**Comment by dougli1sqrd** [comment]
@monicacecilia I can't add you as a reviewer apparently, so I "assigned" instead :)

File: public/team.yaml [hunk 1]
```diff
@@ -111,9 +111,10 @@ institutions:
         alumni: false
         bio: Seth is an occasional software engineer for the Monarch Initiative. He specializes in tools for science and education and has participated in projects in a broad range of fields, including Bioinformatics, Planetary Science, Geology, and Linguistics. His current focus is on capturing and making use of ontology and annotation data. His educational background is undergraduate degrees in Mathematics and Computer Science (UCSC).
       - name: Eric Douglass
-        picture: img/team/person-unknown.png
+        picture: img/team/EricDouglass1.jpg
         title: Software Engineer
         alumni: false
+        bio: Eric is a software engineer that works with Seth on Chris on data validation pipelines, Turtle and Ontologies in triplestores and other tools. He comes from a background originally in physics from Lewis and Clark college and then software development in industry, Eric got his first taste of ontologies at his first tech job in Portland.
```
Comment (monicacecilia) [inline public/team.yaml]: Correct typo: perhaps it should be 'works with Seth ***and*** Chris on data validation pipelines?

Comment (dougli1sqrd) [reply public/team.yaml]: ah yes haha

Comment (monicacecilia) [reply public/team.yaml]: brilliant.

**Pull request merged**